### ISSUE TITLE
I can't delete new file

### ISSUE THREAD

**Issue by qupan** [robotframework/RIDE#1818]
Hello,I have a problem,I create a new file,But I faild to delete this new file.
The file still exsts.Can you solve this problem.Thank you very much.

**Comment by Nyral** [comment]
@HelioGuilherme66 Error occurs if you try to delete a newly created suite.
```
  File "C:\Python37\lib\site-packages\robotide\ui\treenodehandlers.py", line 480, in OnDelete
    self.controller.execute(DeleteFile())
  File "C:\Python37\lib\site-packages\robotide\controller\basecontroller.py", line 27, in execute
    return command.execute(self)
  File "C:\Python37\lib\site-packages\robotide\controller\ctrlcommands.py", line 489, in execute
    context.remove_from_filesystem()
  File "C:\Python37\lib\site-packages\robotide\controller\filecontrollers.py", line 320, in remove_from_filesystem
    if os.path.exists(path):
  File "C:\Python37\lib\genericpath.py", line 19, in exists
    os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType
```

**Comment by HelioGuilherme66** [comment]
Fixed by #1950 and #1951
